### ISSUE TITLE
How do I delete an assignment?

### ISSUE THREAD

**Issue by das-g** [okpy/ok#1365]
According to #465, there should be a way in the web interface to delete an assignment.

According to https://github.com/okpy/ok/wiki/Permissions#updatedelete-1 this should be possible if an assignment has no submissions.

However, I can't seem to figure out where or what to click to do that.

If I would like to delete assignment "Hog" in the following example, how would I do it through the web interface? (I'm logged in as the example staff member who created the assignment and the assignment doesn't have any submissions, yet.)

![assignments overview](https://user-images.githubusercontent.com/97746/60274116-ca4e8b80-98f7-11e9-9ce2-f05d359710d8.png)

![assignment details / settings](https://user-images.githubusercontent.com/97746/60274182-ebaf7780-98f7-11e9-81db-b55c1afcb12e.png)

**Comment by Sumukh** [comment]
I don't think we actually support deletions on our latest code base (the issues/documentation you found seem related to Ok v2 - and we did a total rewrite) 

For now, maybe a rename & uncheck on "Visible on student dashboard" should be somewhat close in terms of functionality
